### PR TITLE
add chevrotain as dependency to VRMLLoader module

### PR DESCRIPTION
**Pull request by palortoff**

### CODE REVIEW

File: examples/jsm/loaders/VRMLLoader.js [hunk 1]
```diff
@@ -34,6 +34,7 @@ import {
 	Vector3,
 	VertexColors
 } from "../../../build/three.module.js";
+import chevrotain from "../libs/chevrotain.min.js";
 
 /* global chevrotain */
```
Comment (gkjohnson) [inline examples/jsm/loaders/VRMLLoader.js]: I think you should be able to remove this line now, as well.

**Comment by Mugen87** [comment]
You can remove the following line since it should be not necessary anymore after your change:

https://github.com/mrdoob/three.js/blob/c11ee3cfd625c556022fe45cbe6ad163940b600a/examples/webgl_loader_vrml.html#L25

File: examples/jsm/loaders/VRMLLoader.js (hunk 1 above)
Comment (Mugen87) [reply examples/jsm/loaders/VRMLLoader.js]: We can do this in a separate PR. Lines line this exists in multiple files.

Comment (palortoff) [reply examples/jsm/loaders/VRMLLoader.js]: `examples/jsm/loaders/VRMLLoader.js` is generated from `examples/js/loaders/VRMLLoader.js` where the comment is still needed. 

We could adapt `utils/modularize.js` to remove the global. Adapting the global line to remove imported globals would make the code unnecessarily complex. We could remove the global comment completely once all modules have their dependencies correctly imported.

**Comment by palortoff** [comment]
Done :)

**Comment by Mugen87** [comment]
Um, it seems the example does not work anymore:

https://raw.githack.com/palortoff/three.js/add-chevrotain-lib/examples/webgl_loader_vrml.html

File: examples/jsm/loaders/VRMLLoader.js (hunk 1 above)
Comment (gkjohnson) [reply examples/jsm/loaders/VRMLLoader.js]: Oh yes I forgot that file is generated :X

Feel free to ignore :) Modifying the script seems overkill for a non functional change. I say keep as is.

**Comment by Mugen87** [comment]
@palortoff Are you aware that libraries needs ES6 builds so they can be imported via ES6 `import` statements?

**Comment by palortoff** [comment]
Sorry, I missed that. Now there is an ES6 export.

File: examples/jsm/libs/chevrotain.min.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
...;if(!1!==(Z=Q.short)?y.charCodeAt(v)===Z&&($=!0):!0===Q.isCustom?$=null!==K.exec(y,v,R,S):(this.updateLastIndex(K,v),$=null!==K.exec(t)),!0===$)break}f=v-X,d=this.config.errorMessageProvider.buildUnexpectedCharactersMessage(y,X,f,z,q),A.push({offset:X,line:z,column:q,length:f,message:d})}}return this.hasCustom||(R.length=g),{tokens:R,groups:S,errors:A}},t.prototype.handleModes=function(t,e,n,r){if(!0===t.pop){var i=t.push;e(r),void 0!==i&&n.call(this,i)}else void 0!==t.push&&n.call(this,t.push)},t.prototype.chopInput=function(t,e){return t.substring(e)},t.prototype.updateLastIndex=function(t,e){t.lastIndex=e},t.prototype.updateTokenEndLineColumnLocation=function(t,e,n,r,i,o,a){var s,u;void 0!==e&&(u=(s=n===a-1)?-1:0,1===r&&!0===s||(t.endLine=i+u,t.endColumn=o-1-u))},t.prototype.computeNewColumn=function(t,e){return t+e},t.prototype.createTokenInstance=function(){for(var t=[],e=0;e<arguments.length;e++)t[e]=arguments[e];return null},t.prototype.createOffsetOnlyToken=function(t,e,n,r){return{image:t,startOffset:e,tokenTypeIdx:n,tokenType:r}},t.prototype.createStartOnlyToken=function(t,e,n,r,i,o){return{image:t,startOffset:e,startLine:i,startColumn:o,tokenTypeIdx:n,tokenType:r}},t.pro
```
Comment (Mugen87) [inline examples/jsm/libs/chevrotain.min.js]: Please use `var` instead of `const`. And also rename the file to `chevrotain.module.js` (see #16862).

Comment (palortoff) [reply examples/jsm/libs/chevrotain.min.js]: Done

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Comment by gkjohnson** [comment]
Maybe it's worth putting an issue into the chevrotain repo to provide an es6 module so it doesn't have to be modified manually?

**Comment by Mugen87** [comment]
Done: https://github.com/SAP/chevrotain/issues/976

**Pull request merged**